### ISSUE TITLE
Identifying the relevance of new articles to COVID-19 NPIs

### ISSUE THREAD

**Issue by jajsmith** [jajsmith/COVID19NonPharmaceuticalInterventions#12]
With the introduction of automated article retrieval, we would like to make it easier to identify which articles are relevant to people doing intervention labeling. As more and more articles have nothing to do with COVID-19, we would like to filter these out, or at least predict the most relevant ones.

To do this, lets try two different approaches:
1. Baseline approach - just filter articles that contain a few relevant keywords (eg. "COVID-19", "Coronavirus", "public health").
2. Use topic modeling on intervention articles and predict the likelihood of a new being about covid.

Some more description of the second approach:
1. Build a dataset that mixes the labeled sources (from the latest CAN-NPI release) with the retrieved articles from the autosource branch. Label articles with interventions as being about COVID-19 and the rest as not. May be able to retrieve more historical articles and label them as not COVID-19 (eg. from same time frame in 2019) to balance the data. 
2. Split the data into a training set and test set (by article and by time) and then 
3. Run topic modeling on the training set and use the topic weightings as inputs to the model
4. Train any number of models (try logistic, decision tree, SVM from scikit-learn) to try and learn their COVID-19 relevance. 
5. Evaluate on the test set. Determine good evaluation metrics for this task
6. Compare results with the baseline filtering.

Notes:
- will need to to treat french and english language articles separately - compare performance on these two languages
- get baseline filtering working first and report accuracy
- report on whether this could be extended to determine content of documents as it relates to specific interventions (Eg. start with Oxford financial labels E1-E4)
